run-tests.php

```diff
@@ -3719,4 +3595,49 @@ function check_proc_open_function_exists()
     }
 }
 
+
+
+
+function getNumberOfWorkers($workers): int
```

Comment:
The file is already in `snake_case`, we should keep that convention for consistency, same for the other functions added below and some of the variables within them

Reply:
resolved. everything is snake case.